lib/datastore/request.js

```diff
@@ -465,17 +460,13 @@ DatastoreRequest.prototype.delete = function(keys, callback) {
  *     stream instance is returned.
  *
  * @example
- * //-
- * // Where you see `transaction`, assume this is the context that's relevant to
- * // your use, whether that be a Dataset or a Transaction object.
- * //-
  *
  * // Retrieve 5 companies.
- * transaction.runQuery(query, function(err, entities, endCursor, apiResponse) {
+ * dataset.runQuery(query, function(err, entities, endCursor, apiResponse) {
```

Comment:
You're right JJ, this is likely because the sandbox is shared between files meaning that any values defined in one are accessible in another. I will update the test to use a new sandbox for each file as long as it's relatively easy to migrate our existing code over.